src/doc/examples/readme.md

```diff
@@ -0,0 +1,8 @@
+# Examples
+
+List with examples, how can be lockable.resources used.
+
+When you has more question or some other use cases please create new issue with label 'documentation'.
```

Comment:
I think the reader will understand this better if it is moved after the list of examples.  They read the opening paragraph and review the items on the list.  If the items in the list solve their issue, they are done.  Otherwise, they want to know how to request additional examples or want to offer additional examples.
```suggestion
If you have an example to share, please create a [new documentation issue](https://github.com/jenkinsci/lockable-resources-plugin/issues/new?assignees=&labels=documentation&template=3-documentation.yml) and provide additional examples as a [pull request](https://github.com/jenkinsci/lockable-resources-plugin/pulls) to the repository.
If you have a question, please open a [GitHub issue](https://github.com/jenkinsci/lockable-resources-plugin/issues/new/choose) with your question.
```